_data/identity.yml

```
@@ -86,7 +86,10 @@ websites:
     tfa:
       - totp
       - hardware
-    doc: https://helpdesk.lastpass.com/multifactor-authentication-options/
+      - proprietary
+      - u2f
+    doc: https://support.logmeininc.com/lastpass/help/enable-multifactor-authentication-lp010002
+    exception: "Duo, RSA SecurID, Symantec VIP, and SecureAuth require LastPass Enterprise or Identity"
```

Such a specific exception text is bound to be outdated rather easily. I'd go for a more broad one.
```suggestion
    exception: "Availability of 2FA options depends on your account type."
```